sklearn/pipeline.py

```diff
@@ -481,7 +481,7 @@ def _inverse_transform(self, X):
         return Xt
 
     @if_delegate_has_method(delegate='_final_estimator')
-    def score(self, X, y=None):
+    def score(self, X, y=None, **score_params):
```

Comment:
I'd be tempted to be more conservative and make this only about `sample_weight`. Is there no prospect of allowing other kwargs with the `__` magic to be passed to the transformers along the way?